config/exchanges/CA-NB_US-NE-ISNE.yaml

```diff
@@ -2,5 +2,5 @@ lonlat:
   - -67.7771
   - 46.139
 parsers:
-  exchange: CA_NB.fetch_exchange
```

Comment:
This parser don't support fetching historical data but US_NEISO does, so let's use that instead and re-fetch as much data we can when it's merged.